Dockerfile

```diff
@@ -1,23 +1,32 @@
-FROM swift:5.1
+FROM swift:5.1 as build
 
 RUN apt-get update
-RUN apt-get install -y cmake libpq-dev libssl-dev libz-dev openssl postgresql
+RUN apt-get install -y cmake libpq-dev libssl-dev libz-dev openssl
 
-WORKDIR /app
+WORKDIR /build
 
-COPY Makefile ./
-COPY Package.swift ./
+COPY Package.swift .
 COPY Sources ./Sources
 COPY Tests ./Tests
 
-# cmark
-RUN git clone https://github.com/commonmark/cmark && \
-	cd cmark && \
-	git checkout 1880e6535e335f143f9547494def01c13f2f331b
+RUN git clone https://github.com/commonmark/cmark \
+  && cd cmark \
+  && git checkout 1880e6535e335f143f9547494def01c13f2f331b
 RUN make -C cmark INSTALL_PREFIX=/usr
 RUN make -C cmark install
 
-RUN swift package update
-RUN swift build --product Server --configuration release -Xswiftc -g && \
-    swift build --product Runner --configuration release -Xswiftc -g
```

Comment:
Do you know what `-g` was doing here?

Reply:
Yeah that is in order to include the stack trace symbols so that we get crash logs on linux:

https://github.com/pointfreeco/pointfreeco/pull/459/files#r312045748